storage/redis/peer_store.go

```diff
@@ -263,32 +279,145 @@ type peerStore struct {
 	cfg Config
 	rb  *redisBackend
 
+	scripts scripts
+
 	closed chan struct{}
 	wg     sync.WaitGroup
 }
 
-func (ps *peerStore) groups() []string {
-	return []string{bittorrent.IPv4.String(), bittorrent.IPv6.String()}
+func addressFamilies() []bittorrent.AddressFamily {
+	return []bittorrent.AddressFamily{bittorrent.IPv4, bittorrent.IPv6}
 }
 
-func (ps *peerStore) leecherInfohashKey(af, ih string) string {
-	return af + "_L_" + ih
+// Constants used for constructing/deconstructing swarm keys.
+const (
+	swarmPrefix                = "swarm_"
+	seedersSuffix              = "_seeders"
+	leechersSuffix             = "_leechers"
```

Comment:
Can we go back to using the smallest constants possible here? Keys like this are readable, but are going to eat memory.

Reply:
I have now boiled this down to the shortest version possible, I believe.